README.md

```diff
@@ -73,10 +73,17 @@ pg_ctl start -D data
 elasticmq& # queues are preconfigured, so no additional work needed
 ```
 
+With `pg_ctl` the PostgreSQL server will keep running until you restart your system. 
+On the other hand ElasticMQ (`elasticmq`) will stop when you close the nix shell, 
+it's only needed if you want to work on asynchronous/worker-related tasks. 
+
 Once you've completed these steps, you should be able to clone the [API
 repository](https://github.com/ca-la/api) and run `bin/boostrap` to set up the
 required databases, insert pricing data, etc.
 
+To avoid running `elasticmq`, generally you can start just the API without the worker. 
```

Comment:
Keeping it here too is fine with me, but I feel like this would benefit from being added to the API readme since that's where the script lives - think you could add it there too @gabrielmadruga ?

Reply:
Will do.